docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -23,8 +23,13 @@ NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
 * View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, then click on a rule name in the All rules table to view the alerts associated with the rule.
-* Visualize alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
-* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the Count table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
+* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the *Count* table. For example, if you select `signal.rule.name`, the table displays the number of alerts generated by each rule.
```

Comment:
Guessing the re-order was because of the reordering of the items in the view? Was just curious

Reply:
Yep! Just matching the order of the items on the page, top to bottom, left to right.